zmon_aws_agent/postgresql.py

```diff
@@ -8,27 +8,41 @@
 from zmon_aws_agent.aws import entity_id
 from zmon_aws_agent.common import call_and_retry
 
+from opentracing_utils import trace, get_span_from_kwargs
```

Comment:
With latest opentracing-utils we can use ``extract_span_from_kwargs`` iso ``get_span_from_kwargs``